src/PeriodEnd/ProviderAdjustments/DeployProviderAdjustments/sql/dml/07 PeriodEnd.ProviderAdjustments.Populate.Reference.History.dml.sql

```diff
@@ -13,18 +13,35 @@ INSERT INTO [Reference].[ProviderAdjustmentsHistory] (
         [CollectionPeriodMonth],
         [CollectionPeriodYear]
     )
-    SELECT
-        [Ukprn],
-        [SubmissionId],
-        [SubmissionCollectionPeriod],
-        [SubmissionAcademicYear],
-        [PaymentType],
-        [PaymentTypeName],
-        [Amount],
-        [CollectionPeriodName],
-        [CollectionPeriodMonth],
-        [CollectionPeriodYear]
-    FROM ${DAS_PeriodEnd.FQ}.ProviderAdjustments.Payments
-    WHERE Ukprn IN (SELECT Ukprn FROM Reference.ProviderAdjustmentsProviders)
-        AND SubmissionAcademicYear = ${YearOfCollection}
+SELECT 
+	[Ukprn],
+    [SubmissionId],
+    [SubmissionCollectionPeriod],
+    [SubmissionAcademicYear],
+    [PaymentType],
+    [PaymentTypeName],
+    [Amount],
+    [CollectionPeriodName],
+    [CollectionPeriodMonth],
+    [CollectionPeriodYear]
+FROM OPENQUERY(${DAS_PeriodEnd.servername}, '
+		SELECT
+			[Ukprn],
+			[SubmissionId],
+			[SubmissionCollectionPeriod],
+			[SubmissionAcademicYear],
+			[PaymentType],
+			[PaymentTypeName],
+			[Amount],
+			[CollectionPeriodName],
+			[CollectionPeriodMonth],
+			[CollectionPeriodYear]
+		FROM 
+			${DAS_PeriodEnd.databasename}.ProviderAdjustments.Payments'
+    ) AS r
+WHERE Ukprn IN (
+        SELECT Ukprn
+        FROM Reference.ProviderAdjustmentsProviders
+      )
+      AND SubmissionAcademicYear = ${YearOfCollection}
```

Comment:
Again, looks like the SubmissionAcademicYear = ${YearOfCollection} could be inside or outside the OPENQUERY.